src/views/HomeView.vue

```diff
@@ -14,12 +14,19 @@ const user = reactive<User>({
   selectLang: "",
   selectLevel: "",
 });
+const headerRef = ref<InstanceType<typeof HeaderAll>>();
+let homeHeightStyle = ref<string>();
+onMounted(() => {
+  const headerHeight = ref<number>(headerRef.value.$el.clientHeight);
+  const headerHeightVh = ref<number>((headerHeight.value * 100) / 1000);
```

Comment:
headerの高さの求め方は以下です
今、viewsフォルダの各ファイルに HeaderAll を指定しています。
App.vueに集約した時に式の変更をお願いします
```suggestion
  const headerHeightVh = ref<number>((headerHeight.value / '画面全体高さ") * 100);
```